### ISSUE TITLE
AriesFrameworkError: Missing required dependency: 'Wallet'

### ISSUE THREAD

**Issue by realabbas** [hyperledger/aries-framework-javascript#1476]
Hey, I am trying to use `@hyperledger/aries-askar-react-native` with the following configuration, please have a look

### Versions

```
  "@aries-framework/anoncreds": "^0.4.0-alpha.127",
    "@aries-framework/anoncreds-rs": "^0.4.0-alpha.127",
    "@aries-framework/askar": "^0.4.0-alpha.127",
    "@aries-framework/indy-vdr": "^0.4.0-alpha.127",
    "@aries-framework/react-native": "^0.4.0-alpha.127",
    "@hyperledger/anoncreds-react-native": "^0.1.0-dev.18",
    "@hyperledger/aries-askar-react-native": "^0.1.0-dev.11",
    "@hyperledger/indy-vdr-react-native": "^0.1.0-dev.15",
    "@aries-framework/core": "0.4.0-alpha.127",
    "react": "17.0.2",
    "react-native": "0.66.1",
    "react-native-fs": "^2.16.6",
    "react-native-get-random-values": "^1.8.0"
```

I am getting the following error while initialising the agent

```
[AriesFrameworkError: Missing required dependency: 'Wallet'. You can register it using one of the provided modules such as the AskarModule or the IndySdkModule, or implement your own.]
```

Code

```
const init = async () => {
    console.log("INIT CALLED")
    const agent = new Agent({
      config: {
        label: 'testing-askar',
        walletConfig: {
          id: 'alice',
          key: 'pw',
        },
        // logger: new ConsoleLogger(LogLevel.debug),
      },
      dependencies: agentDependencies,
      modules: {
        askar: new AskarModule({
          ariesAskar,
        }),
        mediationRecipient: new MediationRecipientModule({
          mediatorPickupStrategy: MediatorPickupStrategy.PickUpV2,
          mediatorInvitationUrl:c 
        }),
        credentials: new CredentialsModule({
          autoAcceptCredentials: AutoAcceptCredential.ContentApproved,
          credentialProtocols: [
            new V2CredentialProtocol({
              credentialFormats: [new LegacyIndyCredentialFormatService()],
            }),
          ],
        }),
        proofs: new ProofsModule({
          autoAcceptProofs: AutoAcceptProof.ContentApproved,
          proofProtocols: [
            new V1ProofProtocol({
              indyProofFormat: new LegacyIndyProofFormatService(),
            }),
            new V2ProofProtocol({
              proofFormats: [new LegacyIndyProofFormatService()],
            }),
          ],
        }),
        anoncreds: new AnonCredsModule({
          registries: [new IndyVdrAnonCredsRegistry()],
        }),
        anoncredsRs: new AnonCredsRsModule({
          anoncreds: anoncreds,
        }),
        indyVdr: new IndyVdrModule({
          indyVdr: indyVdr,
          networks: [
            {
              genesisTransactions: trential,
              indyNamespace: 'trential-dev',
              isProduction: false,
            },
          ],
        }),
      },
    });
    const httpOutboundTransport = new HttpOutboundTransport();
    const wsOutboundTransport = new WsOutboundTransport();
    agent.registerOutboundTransport(wsOutboundTransport);
    agent.registerOutboundTransport(httpOutboundTransport);
    await agent.initialize();
  };
```

**Comment by TimoGlastra** [comment]
@realabbas we just released AFJ 0.4.0 (stable release). Could you try updating all `@aries-framework` versions to `^0.4.0`, and all `@hyperledger` versions to `^0.1.0`

**Comment by blu3beri** [comment]
Closing due to presumed fix by v0.4.0 release.
